src/main/java/com/google/api/codegen/transformer/java/JavaGapicSurfaceTransformer.java

```diff
@@ -249,7 +243,8 @@ private void addXSettingsImports(SurfaceTransformerContext context) {
     ModelTypeTable typeTable = context.getTypeTable();
     typeTable.saveNicknameFor("com.google.api.gax.core.ConnectionSettings");
     typeTable.saveNicknameFor("com.google.api.gax.core.RetrySettings");
-    typeTable.saveNicknameFor("com.google.api.gax.grpc.ApiCallSettings");
+    typeTable.saveNicknameFor("com.google.api.gax.grpc.UnaryApiCallSettings");
+    typeTable.saveNicknameFor("com.google.api.gax.grpc.StreamingCallSettings");
```

Comment:
Right, I was thinking we now have the import cleaner so I just did not implement the import logic. The issue is that the branch that generates the code is not language specific. (e.g. StreamingCallSettings are generated in ApiCallableTransformer). 

My fix is to add an extra helper method to determine we need to add such imports.